backend/session/migrations/0003_auto_20231123_1339.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.20 on 2023-11-23 12:39
+
+from django.db import migrations
+import json
+
+def forwards_func(apps, schema_editor):
+    Session = apps.get_model('session', 'Session')
+    for session in Session.objects.all():
+        if session.json_data:
+            for key,value in json.loads(session.json_data).items():
+                session.json_temp[key] = value
+                session.save()
+                
+def reverse_func(apps, schema_editor):
```

Comment:
I believe you'll also need to use the `apps.get_model` mechanism for the reverse data migration.

Reply:
Good catch !